.github/renovate.json5

```diff
@@ -74,6 +74,10 @@
         "go.mod",
         "go.sum"
       ],
+      "postUpdateOptions": [
+	// update source import paths on major updates
+        "gomodUpdateImportPaths",
+      ]
```

Comment:
@tklauser @aanm Note, there is a missing `,`, c.f. https://github.com/cilium/cilium/issues/23230

Reply:
Whoopsie, opened #24109 to fix it. Thanks for the report.